src/components/checkbox/styled-components.js

```diff
@@ -163,6 +202,13 @@ export const Checkmark = styled('span', props => {
     marginLeft: $theme.sizing.scale200,
     marginRight: $theme.sizing.scale200,
   };
+};
+
+export const Checkmark = styled('span', props => {
+  const {$checkMarkType} = props;
```

Comment:
Should we be consistent with naming? The styled checkmark export is `Checkmark`, but the checkmark type is `checkMarkType`. Can we use `checkmarkType` instead?

Reply:
ok, sure